requirements.txt

```diff
@@ -1 +1,2 @@
+paramiko==2.6.0
```

Comment:
@john-dupuy Do you see this causing any issues with Shinken? Can you please snapshot the Shinken VM and then try to install Paramiko before we merge this PR please?

Reply:
Paramiko is currently installed in Shinken but it is version 2.2.3. Do we need the more recent version?

Reply:
I think it should be fine, @lina-nikiforova can help here more. But I do think if there is a possibility to upgrade without breaking stuff, we should attempt to upgrade. What you both think?

Reply:
Yeah, I can try and see, I don't think it will break anything. It depends on the `linux-ssh-checks`, since those are the ones that currently use Paramiko.

Reply:
@john-dupuy I haven't tried another version. Please try and if we really can't upgrade, I will check whatever version works

Reply:
Thanks @lina-nikiforova, just updated shinken to use paramiko 2.6.0 and everything appears to be running okay.